site/weave-docker-api.md

```diff
@@ -10,6 +10,21 @@ network when they are started using the ordinary Docker
 or the [remote API](https://docs.docker.com/reference/api/docker_remote_api/),
 instead of `weave run`.
 
+###Attaching Containers to a Weave Network
+
+There are two ways to attach containers to a Weave network, 
+the first uses the Weave Net Docker API Proxy, and is described in this section.  
+The other method uses the Docker Network Plugin framework. It’s your choice which method to use. 
+
+To use the Docker Network Plugin, launch Weave with the --net flag.  Note that after 
+you’ve launched Weave Net with this flag, then the Weave Docker API Proxy is automatically disabled.  
```

Comment:
Not sure. The *intention* expressed by `--net blah` and `WEAVE_CIDR=none` is quite different. In the former case the proxy "gets out of the way" since a user specifying `--net blah` probably wants to use Docker networking (including, but not limited to, using weave via docker networking, through the plugin). In the latter, the user explicitly *instructs* the proxy to get out of the way.